src/components/TeamForm/TeamForm.js

```diff
@@ -28,20 +30,26 @@ class TeamForm extends Component {
     };
     this.state = {
       address: props.team.address,
-      name: props.team.name
+      name: props.team.name,
+      voteDuration: props.team.vote_duration
     };
   }
 
   handleChange = field => event => this.setState({ [field]: event.target.value });
 
+  handleKeyPress = field => event => field === 'voteDuration' ? onlyAllowNonZeroDigits(event) : null;
```

Comment:
Could handle this by using an HTML 5 `number` input field with the proper params, but thought this might be compatible with more browsers.

Reply:
Any browser that supports React supports the `number` input field - I'd probably use that instead, as it includes built-in validation to prevent non-numerical characters. It won't prevent you from entering them, but prevents the form from being submitted, which is good enough to me.

Reply:
Ok, I'll make the switch to the `number` input field and remove `formValidations.js`.  Just so I know for the future, would this have been the right pattern to follow (i.e. moving something like that into a helper module and importing it this way)?

Reply:
Yeah, that seems like a good way of doing things, although I would hesitate to over-engineer and abstract away too many methods that might not ever be used anywhere else. In this case I'd be okay with either approach.